quadriga/Docker/mysql/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mysql:9.0.0
+
+ENV MYSQL_ROOT_PASSWORD=root
+ENV MYSQL_DATABASE=quadriga
+ENV MYSQL_USER=quadriga
+ENV MYSQL_PASSWORD=quadriga
+
+
+COPY my.cnf /etc/mysql/conf.d/my.cnf
```

Comment:
where is this file? and do we really need a dockerfile just for mysql? why not use the image?